content/blog/2022/08/2022-08-24-jenkinsfile-runner-as-github-actions.adoc

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title: "A near Feature-complete version of Jenkinsfile Runner Actions as GitHub Actions"
+tags:
+- gsoc
+- gsoc2022
+- jenkinsfile-runner
+- github-actions
+author: YimingGong
+---
+
+== Abstract
+
+Jenkinsfile Runner Action for GitHub Actions aims at providing customized environments and useful GitHub Actions for the users.
+Therefore, users can execute the Jenkins pipeline in the GitHub Actions runners.
+It wants to run Jenkins pipeline in a Function-as-a-Service context.
```

Comment:
```suggestion
It enables running Jenkins pipelines in a Function-as-a-Service context.
```

Probably also link to https://en.wikipedia.org/wiki/Function_as_a_Service so that people not familiar with FaaS have a better understanding.